contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -11,21 +13,31 @@
 ALL_PACKS = '*'
 PACK_REPO_ROOT = 'packs'
 MANIFEST_FILE = 'pack.yaml'
+CONFIG_FILE = 'config.yaml'
+GITINFO_FILE = '.gitinfo'
 PACK_RESERVE_CHARACTER = '.'
 
 
 class InstallGitRepoAction(Action):
-    def run(self, packs, repo_url, abs_repo_base, verifyssl=True, branch='master'):
-        repo_name = repo_url[repo_url.rfind('/') + 1: repo_url.rfind('.')]
+    def run(self, packs, repo_url, abs_repo_base, verifyssl=True, branch='master', subtree=False):
```

Comment:
Hm, I thought this changes were in master before? Or we forgot to merge feature and version branch into master?